### PR TITLE
fix: 404 error when displaying anomaly score graphic

### PR DESCRIPTION
**Pull request by EsadCetiner**
## Proposed changes

Change image location for the anomaly scoring diagram to `as_inbound.svg` to display the image correctly.

### CODE REVIEW

File: content/concepts/anomaly_scoring.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Individual rules designed to detect specific types of attacks and malicious beha
 
 Once all of the rules that inspect *request* data have been executed, *blocking evaluation* takes place. If the anomaly score is greater than or equal to the inbound anomaly score threshold then the transaction is *denied*. Transactions that are not denied continue on their journey.
 
-![Diagram showing an example where the inbound anomaly score threshold is set to 5. A first example request accumulates an anomaly score of 2 and is allowed to pass at the blocking evaluation step. A second example request accumulates an anomaly score of 7 and is denied at the blocking evaluation step.](anomaly_scoring/as_inbound_no_fonts.svg?height=36em)
+![Diagram showing an example where the inbound anomaly score threshold is set to 5. A first example request accumulates an anomaly score of 2 and is allowed to pass at the blocking evaluation step. A second example request accumulates an anomaly score of 7 and is denied at the blocking evaluation step.](as_inbound.svg)
```
Comment (theseion) [inline content/concepts/anomaly_scoring.md:22]: When I look at the current file on GitHub, the image is displayed correctly. With your change it is not.

**Review by theseion — CHANGES_REQUESTED** [review]

File: content/concepts/anomaly_scoring.md (hunk 1 above)
Comment (EsadCetiner) [reply content/concepts/anomaly_scoring.md:22]: Where? when I look at https://coreruleset.org/docs/concepts/anomaly_scoring/ or my local hugo instance, the image is missing.

Comment (theseion) [reply content/concepts/anomaly_scoring.md:22]: Hm, that's annoying. It's rendered correctly on GitHub: https://github.com/coreruleset/documentation/blob/main/content/concepts/anomaly_scoring.md.

**Review by theseion — CHANGES_REQUESTED** [review]

**Review by theseion — APPROVED** [review]

**Comment by theseion** [comment]
Oh yes, I remember. The link checker was failing, which is why I had made that change. @fzipi why does Hugo magically resolve that image in the subfolder but refuses to load it when referenced correctly as `anomaly_scoring/as_inbound_no_fonts.svg?height=36em`? If we could use that, the image would be rendered correctly on GitHub and the link checker would be happy.

**Comment by fzipi** [comment]
Need to review this one.

**Comment by theseion** [comment]
Oh, I see. Thanks!

**Pull request merged**